01week/rockPaperScissors.js

```diff
@@ -9,6 +9,44 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  // there are two players, I'll call them hand1 and hand2
+
+
+  // if hand1 is rock and hand2 is paper then hand2 wins
+  if (hand1.toLowerCase().replace(/ /g, '') === 'rock' && hand2.toLowerCase().replace(/ /g, '') = 'paper') {
+    return "Hand two wins!"
+  } else
+
+
```

Comment:
you are only using one '=' in your conditional statement. you need to be using three .